extractor/src/main/java/org/schabi/newpipe/extractor/kiosk/KioskList.java

```diff
@@ -78,10 +79,10 @@ public KioskExtractor getDefaultKioskExtractor(final Page nextPage,
         if (!isNullOrEmpty(defaultKiosk)) {
             return getExtractorById(defaultKiosk, nextPage, localization);
         } else {
-            if (!kioskList.isEmpty()) {
+            final String first = kioskList.keySet().stream().findFirst().orElse(null);
```

Comment:
Isn't `findAny` more appropriate, since you are accessing an unordered set?

Reply:
Sure.